packages/caver-klay/caver-klay-kct/contract/README.md

```diff
@@ -0,0 +1,6 @@
+caver-klay-kct/contract
+=================
+
+Some files in `caver-klay-kct/contract` were derived from [openzeppelin](https://github.com/OpenZeppelin/openzeppelin-contracts).
```

Comment:
Could you please write the version of openzeppelin code?

Reply:
Refer this.
https://github.com/klaytn/klaytn/tree/dev/contracts/externals

Reply:
@KimKyungup Yes, I wrote the version together. :)